apps/middleman/src/app/(landing)/page.tsx

```diff
@@ -1,8 +1,17 @@
+import type { Metadata } from 'next'
 import Hero from "./components/Hero";
 import About from "./components/About";
 import EmptySpace from "./components/EmptySpace";
-import Services from "./components/Services";
-import Footer from "@/app/(landing)/components/Footer";
+import { GetAppName } from '@/actions/ApplicationSettings'
+
+export async function generateMetadata(): Promise<Metadata> {
+  const appName = await GetAppName()
+
+  return {
+    title: `Home - ${appName}`,
+    description: "Light up your earnings with Igniter",
```

Comment:
Maybe we want to use the app name here as well?

```suggestion
    description: `Light up your earnings with ${appName}`,
```